src/plugins/maps_legacy/public/common/opensearch_maps_client.js

```diff
@@ -21,7 +21,7 @@ export class OpenSearchMapsClient extends EMSClient {
       result = await this._fetchWithTimeout(this._manifestServiceUrl);
     } catch (e) {
       // silently ignoring the exception and returning false.
```

Comment:
should this comment be updated since the return value is changed?

Reply:
thanks, just updated.